src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/hosts/host-form/host-form.component.ts

```diff
@@ -47,14 +45,18 @@ export class HostFormComponent extends CdForm implements OnInit {
     private hostService: HostService,
     private taskWrapper: TaskWrapperService,
     private route: ActivatedRoute,
-    private location: Location
+
+    @Inject('hideMaintenance') @Optional() public hideMaintenance?: boolean
   ) {
     super();
     this.resource = $localize`host`;
     this.action = this.actionLabels.ADD;
   }
 
   ngOnInit() {
+    if (this.router.url.includes('hosts')) {
```

Comment:
Lets have a constant for hosts and then reuse

Reply:
done